Dockerfile-dev

```diff
@@ -0,0 +1,16 @@
+FROM python:3
+
+WORKDIR /home/app
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY .env .
+COPY . .
```

Comment:
we should try to avoid using `COPY . .`. Let's be more specific and just copy what we need 🙂